dji_robomaster/agent/robomaster_mover.py

```diff
@@ -0,0 +1,286 @@
+"""
+Copyright (c) Facebook, Inc. and its affiliates.
+"""
+import logging
+import numpy as np
+import cv2
+import os
+import math
+import copy
+import robomaster
+from robomaster import robot
+from perception import RGBDepth
+import Pyro4
+from Pyro4 import naming
+from scipy.spatial.transform import Rotation
+Pyro4.config.SERIALIZER = "pickle"
+Pyro4.config.SERIALIZERS_ACCEPTED.add("pickle")
+
+
+#from perception import RGBDepth
+#from objects import Marker, Pos
+
+
+#FIXME this is exact copy from locobot utils, don't copy
+def transform_pose(XYZ, current_pose):
+    """
+    Transforms the point cloud into geocentric frame to account for
+    camera position
+    Input:
+        XYZ                     : ...x3
+    current_pose            : camera position (x, y, theta (radians))
+    Output:
+        XYZ : ...x3
+    """
+    R = Rotation.from_euler("Z", current_pose[2]).as_matrix()
+    XYZ = np.matmul(XYZ.reshape(-1, 3), R.T).reshape((-1, 3))
+    XYZ[:, 0] = XYZ[:, 0] + current_pose[0]
+    XYZ[:, 1] = XYZ[:, 1] + current_pose[1]
+    return XYZ
+
+
+def to_interpreter_xzy(xyyaw):
```

Comment:
also add a figure for the dji coordinate axes orientation - something like [this](https://github.com/facebookresearch/droidlet/blob/main/locobot/agent/rotation.py#L55).